src/base/rss/rss_feed.cpp

```diff
@@ -56,19 +56,22 @@
 
 const QString KEY_UID = u"uid"_s;
 const QString KEY_URL = u"url"_s;
+const QString KEY_REFRESHINTERVAL = u"refreshInterval"_s;
 const QString KEY_TITLE = u"title"_s;
 const QString KEY_LASTBUILDDATE = u"lastBuildDate"_s;
 const QString KEY_ISLOADING = u"isLoading"_s;
 const QString KEY_HASERROR = u"hasError"_s;
 const QString KEY_ARTICLES = u"articles"_s;
 
+using namespace std::chrono_literals;
 using namespace RSS;
 
-Feed::Feed(const QUuid &uid, const QString &url, const QString &path, Session *session)
+Feed::Feed(Session *session, const QUuid &uid, const QString &url, const QString &path, std::chrono::seconds refreshInterval)
```

Comment:
You can add `const` here:
```suggestion
Feed::Feed(Session *session, const QUuid &uid, const QString &url, const QString &path, const std::chrono::seconds refreshInterval)
```
And to other similar cases.

Reply:
> You can add `const` here
And to other similar cases.

Ok. Done.